GEMINI.md

```
@@ -636,7 +633,7 @@ class FeedParserService:
 Esta estrutura separa lógica de negócio (services) de acesso a dados (models) e apresentação (views/serializers).
 
 - Django Docs: https://docs.djangoproject.com/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)